### DISCUSSION TITLE
Universal Sharing via WebDav or SMB

### DISCUSSION THREAD

**Discussion by N20Visuals** [photoprism/photoprism#1858] [Ideas]
Hi Everyone,

i've just started using Photoprism and i think it's the best option out there, really like it! Some features are still missing but i've already seen that some of them are in development or on the roadmap.

One thing i didn't find is an option for some kind of universal sharing via WebDav or through an smb share.
Specifically i'm talking about sharing an album created in photoprism to for example an android tv to use those images as a screensaver. I don't know if that would be possible or even valid but i could see that kind of use case for many people. 

just a thought on execution, might be stupid, i'm more of a advanced user / tech enthusiast than an expert: maybe photoprism could create a accessible folder / share which contains symlinks to the actual originals or the .jpg previews so that it wouldn't be that storage intensive for larger albums.. 

(I'm using an app called "photo gallery and screensaver" for my tv to display personal image without using google photos)
maybe that feature already exists and i'm just missing something, if so please point me in the right direction. Thank you very much!

**Comment by lastzero** [comment]
Thank you! Sorry for the late reply. It is already possible to access and share media files via WebDAV (unless you only want to share some specific wallpapers with your TV):
 
https://docs.photoprism.app/user-guide/sync/webdav/

For everything else, you would currently need to run additional Docker containers or install server applications on your operating system. For example, you could put all wallpapers/screensavers in a special folder that you share.

We may implement a [UPnP AV media server](https://en.wikipedia.org/wiki/Universal_Plug_and_Play#Media_server) directly in PhotoPrism at a later time, as we consider this a "nice-to-have" rather than a core feature. For existing UPnP AV servers, see:

https://en.wikipedia.org/wiki/List_of_UPnP_AV_media_servers_and_clients
